planet/cli/orders.py

```diff
@@ -231,36 +233,61 @@ def read_file_json(ctx, param, value):
 @click.pass_context
 @translate_exceptions
 @coro
-@click.option('--name', required=True)
-@click.option('--id',
-              'ids',
-              help='One or more comma-separated item IDs',
-              type=click.STRING,
-              callback=split_list_arg,
-              required=True)
-# @click.option('--ids_from_search',
-#               help='Embedded data search')
+@click.argument('request',
+                required=True,
+                type=click.Path(exists=True,
+                                resolve_path=True,
+                                writable=True,
+                                allow_dash=True))
```

Comment:
@kevinlacaille here's another approach to an argument that can be text or file: https://github.com/mapbox/mercantile/blob/main/mercantile/scripts/__init__.py#L70.